### PR TITLE
fix(notification): follow notification palette theme

### PR DESCRIPTION
**Pull request by Seojun-Park**
Fixes https://github.com/marmelab/react-admin/issues/8510

### CODE REVIEW

**Comment by Seojun-Park** [comment]
- Warning
Screenshot 2022-12-19 at 14 09 13
- Error
Screenshot 2022-12-19 at 14 09 23

**Review by septentrion-730n — APPROVED** [review]

**Comment by septentrion-730n** [comment]
Thanks for your contribution 👍
Could you rebase your PR on master as it fits for the next 4.6.3 release?

**Comment by fzaninotto** [comment]
You must also change all the error notifications that use the `warning` type, such as https://github.com/marmelab/react-admin/blob/bdff90eeb1e3e7012565c695d6cd7bf5d953d5a6/packages/ra-ui-materialui/src/button/ExportButton.tsx#L59

**Comment by Seojun-Park** [comment]
> You must also change all the error notifications that use the `warning` type, such as
> 
> https://github.com/marmelab/react-admin/blob/bdff90eeb1e3e7012565c695d6cd7bf5d953d5a6/packages/ra-ui-materialui/src/button/ExportButton.tsx#L59

Changes
 - changing commit : 9abb7529e218012d2ef2df3abb963e59739a1551
 - changing commit(docs)  : 468e7b5e22cfb15a1fc1fd88a0da5bfee8b1388b

File: docs/useNotify.md [hunk 1]
```diff
@@ -31,8 +31,10 @@ The callback takes 2 arguments:
 Here are more examples of `useNotify` calls: 
 
 ```js
+// notify an error
+notify(`This is an error`, { type: 'error' });
 // notify a warning
-notify(`This is a warning`, { type: 'warning' });
+notify(`This is a warning`, { type: 'error' });
```
Comment (fzaninotto) [inline docs/useNotify.md:37]: this change is incorrect

Comment (Seojun-Park) [reply docs/useNotify.md]: Indeed

It's back to `warning` : 6be244017c22f9eb51f228d5ce4636352635be55

**Comment by Seojun-Park** [comment]
@slax57 
> Thanks for those changes! Although there are still some `type: 'warning'` left at some places in the code. I'll try to push them directly on your branch, and if I can't I'll provide a diff.

Thank you!

**Comment by slax57** [comment]
Thanks!

**Pull request merged**